src/python/pants/option/options_test.py

```diff
@@ -1438,12 +1451,24 @@ def parse_joined_command_line(*args):
                 "--c=[]",
             ).for_global_scope()
 
+        # When some flags have suggestions, but >=1 don't, we both gives suggestions and point to
+        # `help-advanced`.
+        with self.assertRaisesWithMessage(
+            ParseError,
+            (
+                "Unrecognized command line flags on global scope: --aasdf, --config-overide. "
```

Comment:
This might be clearer as:
```
Unrecognized command line flags on global scope: --aasdf, --config-overide. Did you mean:
  --config-overide: [--config-override]
(See `./pants help-advanced` for all available options.)
```
or
```
Unrecognized command line flags on global scope:
  --aasdf
  --config-overide (did you mean --config-override ?)
(See `./pants help-advanced` for all available options.)
```

It also might be useful to always give the "See `./pants help-advanced $optional_scope`" advice, even when we do have potential candidates?

Reply:
I love it. Thanks for the suggestion!